master/buildbot/test/unit/test_status_buildstep.py

```diff
@@ -155,15 +160,15 @@ def __init__(self):
         trigger_step_status = build.addStepWithName('step_2', Trigger)
         build.builder.master.db = StubDB()
         build.brids = [1]
-        build.builder.master.status.getURLForBuild = lambda x, y, friendly_name: {'path':x, 'text': y}
         build.builder.master.status.getFriendlyName = lambda x: None
+        build.builder.master.status.getURLForBuild = lambda x, y, friendly_name: {'path':x, 'text': y}
+        codebases_arg = "?foo=bar"
 
-        self.assertEqual(len(trigger_step_status.urls), 0)
-
-        trigger_step_status.prepare_trigger_links()
+        trigger_links = yield trigger_step_status.prepare_trigger_links(codebases_arg)
 
-        self.assertEqual(len(trigger_step_status.urls), 1)
+        self.assertEqual(len(trigger_links), 1)
```

Comment:
Can we check this link? I mean checking the assertion value of the `trigger_links` element.

Reply:
I can see 1 problem with checking values of trigger_links
1. trigger_links[0] is not a string but is a 3-elements tuple. Data for this tuple comes from stubbed method getURLForBuild (+codebase_arg). And if I start checking values I won't do an unittest for real tuple but stub tuple

Maybe I should check if this tuple has exactly 3 elements instead of checking values?